lib/stacks/athena-stack.ts

```diff
@@ -0,0 +1,50 @@
+import { Construct } from 'constructs';
+
+import { Aws, Duration, Size, Stack, StackProps } from 'aws-cdk-lib';
+import { CfnWorkGroup } from 'aws-cdk-lib/aws-athena';
+
+import { Database } from '@aws-cdk/aws-glue-alpha';
+
+import { PrivateBucket } from '../constructs/s3/private-bucket';
+
+interface AthenaStackProps extends StackProps {
+    workGroupName: string;
+}
+
+/**
+ *
+ */
+export class AthenaStack extends Stack {
+    constructor(scope: Construct, id: string, private readonly props: AthenaStackProps) {
+        super(scope, id, props);
+
+        //  Buckets
+        new PrivateBucket(this, 'TestData', {
+            bucketName: `dbt-athena-test-data-${Aws.REGION}`,
+            lifecycleRules: [{ expiration: Duration.days(7) }],
+        });
+
+        const athenaQueryResultsBucket = new PrivateBucket(this, 'AthenaQueryResults', {
+            bucketName: `dbt-athena-query-results-${Aws.REGION}`,
+            lifecycleRules: [{ expiration: Duration.days(7) }],
+        });
+
+        // Athena
+        new CfnWorkGroup(this, `Workgroup-${this.props.workGroupName}`, {
+            name: this.props.workGroupName,
+            state: 'ENABLED',
+            workGroupConfiguration: {
+                bytesScannedCutoffPerQuery: Size.gibibytes(10).toBytes(),
+                publishCloudWatchMetricsEnabled: true,
+                resultConfiguration: {
+                    outputLocation: `s3://${athenaQueryResultsBucket.bucketName}/${this.props.workGroupName}/`,
+                },
```

Comment:
this is not really necessary, because in the latest adapter versions we use the location from the workgroup only if it's enforced. I prefer that we control that via `profiles.yml` specifying the s3_data_dir and the s3_staging_dir.

Reply:
I think this is ok. The `outputLocation` is not enforced and can be overwritten. It simplifies a bit if we run queries form Athena console.

Reply:
fair enough, then we can use the same workgroup to testing out queries from it

Reply:
Is that related to the warning posted in the docs? https://dbt-athena.github.io/docs/configuration/table-configuration#table-data-location